src/interface/web/app/common/modelSelector.tsx

```diff
@@ -228,8 +230,9 @@ function ModelItem({ model, isSelected, onSelect, onPeek }: ModelItemProps) {
             onSelect={onSelect}
             ref={ref}
             className="data-[selected=true]:bg-muted data-[selected=true]:text-secondary-foreground"
+            disabled={!isActive && model.tier !== "free"}
         >
-            {model.name}
+            {model.name} {model.tier === "standard" && <span className="text-green-500 ml-2">(Pro)</span>}
```

Comment:
Let's be consistent in naming?

```suggestion
            {model.name} {model.tier === "standard" && <span className="text-green-500 ml-2">(Futurist)</span>}
```